dubbo-plugin/dubbo-spring-security/pom.xml

```diff
@@ -88,6 +88,11 @@
       <artifactId>log4j-slf4j-impl</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Delete it because it already existed in this pom. you'd better install Maven Helper plugin if you use Intelij IDEA to ensure what artifacts have been defined in the pom.